File: beagle-ui/src/main/java/br/com/zup/beagleui/framework/widget/layout/Stack.kt
package br.com.zup.beagleui.framework.widget.layout

import br.com.zup.beagleui.framework.widget.core.Flex
import br.com.zup.beagleui.framework.widget.core.Widget

data class Stack(
    val children: List<Widget>,
    val flex: Flex = Flex()
) : Widget

File: Sources/BeagleUI/Sources/Widgets/Layout/Stack.swift
//
//  Stack.swift
//  BeagleUI
//
//  Created by Daniel Tes on 12/09/19.
//  Copyright © 2019 Daniel Tes. All rights reserved.
//

public struct Stack: Widget, HasAppearance {
    
    // MARK: - Public Properties
    
    public let appearance: Appearance?
    public let children: [Widget]
    
    // MARK: - Initialization
    
    public init(
        appearance: Appearance? = nil,
        children: [Widget] = []
    ) {
        self.appearance = appearance
        self.children = children
    }
    
}
